### PR TITLE
refactor _glow.less

### PR DESCRIPTION
**Pull request by Hisime**
Issue:
https://github.com/sButtons/sbuttons/issues/655

### CODE REVIEW

File: src/components/animated/_glow.less [hunk 1]
```diff
@@ -1,88 +1,51 @@
-.glowBorderRule(@color) {
+.glow(@color) {
+  transition: all 0.5s ease;
```
Comment (shahednasser) [inline src/components/animated/_glow.less:2]: Put this in `.glow-btn` like it was to not generate this code many times when compiling to CSS.

**Review by shahednasser — CHANGES_REQUESTED** [review]
Great work but I need you to run `npm run compile` once you're done with the change below for the changes to take effect, then commit the changes in dist

File: src/components/animated/_glow.less (hunk 1 above)
Comment (Hisime) [reply src/components/animated/_glow.less]: fixed it

File: src/components/animated/_glow.less [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
-    border-color: @black;
-  }
-}
-
 .glow-btn {
   border: 2px solid;
   color: @black;
 
+  .glow(@color) {
+    transition: all 0.5s ease;
```
Comment (shahednasser) [inline src/components/animated/_glow.less:6]: Put it outside of `.glow(@color) after `color: @black` then run `npm run compile` again

**Review by shahednasser — CHANGES_REQUESTED** [review]

**Comment by shahednasser** [comment]
Hi @Hisime I will be closing this PR as it is behind now. The `_glow.less` file has been moved to `src/components/styles/_glow.less`. Please work on it there and submit a new PR.

**Pull request closed without merging**